nncf/quantization/algo.py

```diff
@@ -167,7 +167,16 @@ def apply_to(self, target_model: NNCFNetwork) -> NNCFNetwork:
 
         if self._debug_interface is not None:
             target_model.debug_interface.add_interface(self._debug_interface)
+        
+        if self._collect_compression_metrics:
+            self.metric_store = {}
+            metrics = [NetworkQuantizationShareMetric(target_model, self.weights_quantizers, self.non_weights_quantizers)]
+            for m in metrics:
+                m.collect()
+                self.metric_store[m.NAME] = m.get_metric_table()
```

Comment:
It seems that these new entities are never used anywhere in the user code. Will these be integrated with `.statistics()`?